native/src/lib.rs

```diff
@@ -170,7 +170,7 @@ fn bbs_sign(mut cx: FunctionContext) -> JsResult<JsArrayBuffer> {
 /// {
 ///     "publicKey": ArrayBuffer                // The public key
 ///     "signature": ArrayBuffer                // The signature
-///     "messages": [String, String],           // The messages that were signed as strings. They will be SHAKE-256 hashed
+///     "messages": [ArrayBuffer, ArrayBuffer], // The messages that were signed as strings. They will be Blake2b hashed
```

Comment:
```suggestion
///     "messages": [ArrayBuffer, ArrayBuffer], // The messages that were signed as ArrayBuffers. They will be Blake2b hashed
```

Reply:
@tplooker curious if this is correct as is because it was resolved. Could you add clarification? Happy to resolve if so.